types/react-dual-listbox/react-dual-listbox-tests.tsx

```diff
@@ -93,27 +93,31 @@ const optionsChange = (selectedValues: Array<Option<string>>) => {};
 />;
 
 /** Filtering error examples. */
-// @ts-expect-error You can not use filter properties when `canFilter` is not `true`.
 <DualListBox
     options={flatOptions}
     filter={{
         available: flatOptions.map(o => o.value),
         selected: [],
     }}
+    // @ts-expect-error You can not use filter properties when `canFilter` is not `true`.
     onFilterChange={() => {}}
+    // @ts-expect-error You can not use filter properties when `canFilter` is not `true`.
     filterPlaceholder={""}
+    // @ts-expect-error You can not use filter properties when `canFilter` is not `true`.
```

Comment:
Odd change but may be related to removing legacy context from the constructor signature.